Plugins/Repository.cs

```diff
@@ -1,12 +1,9 @@
-using System;
-using JetBrains.Annotations;
-
 namespace NFive.SDK.Core.Plugins
 {
-	/// <summary>
-	/// Repository configuration for a plugin.
-	/// </summary>
-	[PublicAPI]
+    using JetBrains.Annotations;
+    using System;
+
+    [PublicAPI]
```

Comment:
Looks like some spaces got in here and a few other places.

Reply:
I believe this got fixed with another comment fix